pyramid_oereb/lib/readers/real_estate.py

```diff
@@ -74,10 +74,22 @@ def read(self, nb_ident=None, number=None, egrid=None, geometry=None):
             max_NS03=max_NS03
         )
 
+        real_estate_main_page_view_service = ViewServiceRecord(
+            reference_wms=reference_wms,
+            legend_at_web=Config.get_real_estate_main_page_config().get('view_service').get('legend_at_web'),
```

Comment:
Can you call this method (get_real_estate_main_page_config) only once ?

Reply:
Yes, that would make more sense!